File: 268_MissingNumber.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        impostor = 0
        #nums.sort()
    
        for i in range(len(nums)+1):
            if i in nums:
                continue
            else:
                impostor = i
        
    
        return impostor

File: 268_MissingNumber.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int value = 0;
        vector<int>::iterator it;
        
        for(int i = 0; i <= nums.size(); i++)
        {
            it = find(nums.begin(), nums.end(), i);
            if(it == nums.end())
            {
                value = i;
            }
        }
        
        //cout << value << endl;
        
        return value;
    }
};